src/ui/wxWidgets/PasswordSafeSearch.cpp

```diff
@@ -630,12 +635,27 @@ void PasswordSafeSearch::OnChar(wxKeyEvent& event)
         // then normal copy event shall be handled.
         event.Skip();
       }
-      else {
+      else if ((event.GetKeyCode() == wxT('c')) || (event.GetKeyCode() == wxT('C'))) {
```

Comment:
Nit: assigning `event.GetKeyCode()` to a variable would make these if conditions easier to read. Also, there's no need for () around equality checks since `==` has higher precedence than `||`.